internal/config/config.go

```diff
@@ -394,7 +394,8 @@ func peerFromCR(p metallbv1beta2.BGPPeer, passwordSecrets map[string]corev1.Secr
 
 	// keepalive must be lower than holdtime
 	if keepaliveTime > holdTime {
-		return nil, fmt.Errorf("invalid keepaliveTime %q", p.Spec.KeepaliveTime)
+		return nil, fmt.Errorf("invalid keepaliveTime %q must be smaller than holdtime %q",
```

Comment:
the commit says "add unit test", but the core part is the changed message.

Let's reword the commit, with the unit test being a part of what the commit does.